README.md

```diff
@@ -1,18 +1,20 @@
-# Snakemake workflow: Tn5 RNA-Seq Pipeline
+# Snakemake workflow: TM3'-seq Pipeline
 
-Workflow for the Tn5-TagSeq manuscript.
+Workflow for the TM3'-seq manuscript.
 
 The workflow is written using [Snakemake](https://snakemake.readthedocs.io/).
 Dependencies are installed using [Bioconda](https://bioconda.github.io/).
 
 
 ## Overview
 
-This workflow is designed to generate counts of the number of uniquly mapped reads to each gene for each sample provided.
+The workflow was design to streamline the analsys of TM3'seeq data, however, it can be used to process FASTQ files derived form any other RNA-seq protocol once the samples have been demultiplexed. This step might vary depending on the protocol used for library preparation.
```

Comment:
A few typos:
- [ ] design -> designed
- [ ] analsys -> analysis
- [ ] TM3'seeq -> TM3'seq

Suggestion:
> This step might vary depending on the protocol used for library preparation.

The details of demultiplexing may vary depending on the protocol used for library preparation.